test/e2e/apimachinery/aggregator.go

```diff
@@ -368,7 +391,15 @@ func TestSampleAPIServer(f *framework.Framework, image string) {
 		}
 	}
 	framework.ExpectNoError(err, "gave up waiting for apiservice wardle to come up successfully")
+}
 
+// A basic test if the sample-apiserver code from 1.10 and compiled against 1.10
+// will work on the current Aggregator/API-Server.
```

Comment:
This is now older than we promise to support, isn't it? I guess we can fix that in another PR.